plugins/inara.py

```diff
@@ -21,38 +22,41 @@
 import plug
 logger = logging.getLogger(appname)
 
+if TYPE_CHECKING:
+    def _(x):
+        return x
+
 
 _TIMEOUT = 20
-FAKE = ['CQC', 'Training', 'Destination']	# Fake systems that shouldn't be sent to Inara
+FAKE = ('CQC', 'Training', 'Destination')  # Fake systems that shouldn't be sent to Inara
 CREDIT_RATIO = 1.05		# Update credits if they change by 5% over the course of a session
 
 
-this: Any = sys.modules[__name__]	# For holding module globals
+this: Any = sys.modules[__name__]  # For holding module globals
 this.session = requests.Session()
-this.queue = Queue()	# Items to be sent to Inara by worker thread
-this.lastlocation = None	# eventData from the last Commander's Flight Log event
-this.lastship = None	# eventData from the last addCommanderShip or setCommanderShip event
+this.queue = Queue()  # Items to be sent to Inara by worker thread
+this.lastlocation = None  # eventData from the last Commander's Flight Log event
+this.lastship = None  # eventData from the last addCommanderShip or setCommanderShip event
 
 # Cached Cmdr state
-this.events = []	# Unsent events
-this.cmdr = None
-this.FID = None		# Frontier ID
-this.multicrew = False	# don't send captain's ship info to Inara while on a crew
-this.newuser = False	# just entered API Key - send state immediately
-this.newsession = True	# starting a new session - wait for Cargo event
-this.undocked = False	# just undocked
-this.suppress_docked = False	# Skip initial Docked event if started docked
-this.cargo = None
-this.materials = None
-this.lastcredits = 0	# Send credit update soon after Startup / new game
-this.storedmodules = None
-this.loadout = None
-this.fleet = None
-this.shipswap = False	# just swapped ship
+this.events: List[OrderedDictT[str, Any]] = []  # Unsent events
+this.cmdr: Optional[str] = None
+this.FID: Optional[str] = None		# Frontier ID
+this.multicrew = False  # don't send captain's ship info to Inara while on a crew
```

Comment:
type hints on bools are optional ?

Reply:
They can be inferred more easily by a type checker